src/data/science.json

```
@@ -12,6 +12,7 @@
       "url": "/en/science",
       "external": false
     },
+    "mobileMessage": "This page has been optimized for mobile devices, i.e. graphics are shown as static images.",
```

```suggestion
    "mobileMessage": "This page is optimized for mobile devices, i.e. graphics are shown as static images.",
```

Use present tense "is" as the user is interested in the current state of the page, not the history of the change.